tests/test_config.py

```diff
@@ -71,7 +71,7 @@ def test_global_config_equal_config_defaults() -> None:
 
 def test_config_from_globals_behavior() -> None:
     original_style_option_value = rc.STYLE_OPTION
-    rc.STYLE_OPTION = "new-value"
+    rc.STYLE_OPTION = "bold cyan"
```

Comment:
The reason this is `new-value` and not `bold cyan` is because we want to assert a change occurred. Because `bold cyan` is the default, this change means the test is no longer testing the thing it should be.

Reply:
Thank you for pointing that out and guiding me in the right direction. I revisited my changes, ensured they align with the intended behavior, and have pushed an updated fix. Could you kindly review it and let me know if it works as expected?

Reply:
Thanks for the speedy change! Left a comment in the main thread, tldr I'll either review manually myself end of week, or we can speed this up if you give a reproducible example. Your call.